cli/testflinger_cli/__init__.py

```diff
@@ -328,29 +402,77 @@ def submit(self):
         if self.args.poll:
             self.do_poll(job_id)
 
-    def submit_job_data(self, data):
+    def submit_job_data(self, data: dict):
         """Submit data that was generated or read from a file as a test job"""
         try:
             job_id = self.client.submit_job(data)
         except client.HTTPError as exc:
             if exc.status == 400:
-                raise SystemExit(
+                sys.exit(
                     "The job you submitted contained bad data or "
                     "bad formatting, or did not specify a "
                     "job_queue."
-                ) from exc
+                )
             if exc.status == 404:
-                raise SystemExit(
+                sys.exit(
                     "Received 404 error from server. Are you "
                     "sure this is a testflinger server?"
-                ) from exc
+                )
             # This shouldn't happen, so let's get more information
-            raise SystemExit(
+            sys.exit(
                 "Unexpected error status from testflinger "
-                "server: {}".format(exc.status)
-            ) from exc
+                f"server: {exc.status}"
+            )
         return job_id
 
+    def submit_job_attachments(self, job_id: str, path: Path):
+        """Submit attachments archive for a job to the server
+
+        :param job_id:
+            ID for the test job
+        :param path:
+            The path to the attachment archive
+        """
+        # defaults for retries
+        wait = self.config.get("attachments_retry_wait") or 10
+        timeout = self.config.get("attachments_timeout") or 600
+        tries = self.config.get("attachments_tries") or 3
```

Comment:
It would seem more idiomatic to use the `default` parameter of `dict.get()` for this:

```suggestion
        wait = self.config.get("attachments_retry_wait", 10)
        timeout = self.config.get("attachments_timeout", 600)
        tries = self.config.get("attachments_tries", 3)
```